connexion/apps/flask_app.py

```diff
@@ -85,6 +88,9 @@ def common_error_handler(self, exception):
                 headers=exception.get_headers(),
             )
 
+        if is_server_error:
```

Comment:
Can we replace this with `if response["status"] >= 500`?

Reply:
There is .status_code on the "ConnexionResponse" type.